pkg/webhook/cluster/validation/validation.go

```diff
@@ -247,6 +240,50 @@ func validateClusterNetworkingConfigUpdateImmutability(c, oldC *kubermaticv1.Clu
 	return allErrs
 }
 
+func validateCNIUpdate(cni *kubermaticv1.CNIPluginSettings, oldCni *kubermaticv1.CNIPluginSettings, labels map[string]string) field.ErrorList {
+	specFldPath := field.NewPath("spec")
```

Comment:
`specFieldPath`

Reply:
are you suggesting a different name for the variable?

Reply:
Yes. I thought it was a typo.

Reply:
It's a nit. Not required at all.